io/io/src/RFile.cxx

```diff
@@ -513,3 +530,24 @@ void RFile::Close()
    // NOTE: this also flushes the file internally
    fFile.reset();
 }
+
+std::optional<ROOT::Experimental::RKeyInfo> RFile::GetKeyInfo(std::string_view path) const
+{
+   const TKey *key = GetTKey(path);
+   if (!key)
+      return {};
+
+   RKeyInfo keyInfo;
+   keyInfo.fPath = ReconstructFullKeyPath(*key);
+   keyInfo.fClassName = key->GetClassName();
+   keyInfo.fCycle = key->GetCycle();
+   keyInfo.fTitle = key->GetTitle();
+
+   return keyInfo;
+}
+
+void *ROOT::Experimental::Internal::GetRFileObjectFromKey(RFile &file, const RKeyInfo &key)
```

Comment:
```suggestion
void *ROOT::Experimental::Internal::RFileGetObjectFromKey(RFile &file, const RKeyInfo &key)
```
or just (since `RFile` is already implied from the signature)
```suggestion
void *ROOT::Experimental::Internal::GetObjectFromKey(RFile &file, const RKeyInfo &key)
```
The original reads (to me) as 'Get' 'the RFile Object' (i.e. an object of type `RFile`) `from` `the Key` (as opposed to (I suppose) be meant to read as 'C-Style wrapper around `RFile::GetObjectFromKey` ... so I guess another option would be `RFile__GetObjectFromKey` )